pose_control:
  ros_control:
    rate: 10
    force_cmd_vel: True
  control:
    gains: [0.0 0.0 0.0]
    max_lin_vel: 0.0
    max_ang_vel: 0.0
    lin_threshold: 0.0
    ang_threshold: 0.0